openlibrary/core/helpers.py

```diff
@@ -122,26 +124,32 @@ def days_since(then, now=None):
 
 def datestr(then, now=None, lang=None, relative=True):
     """Internationalized version of web.datestr."""
-    lang = lang or web.ctx.get('lang') or "en"
+    # Get the locale from the cookie or default to 'en'
+    locale = web.cookies(i18n_code='en')
```

Comment:
Should we be tapping into https://docs.python.org/3/library/locale.html ?

Reply:
No. You don't want the server's locale.

Reply:
In any case, this is a good point. I'll go ahead and rename the `locale` variable to avoid any future name-clashing (in case someone imports the `locale` module)